views/pages/HomePage.jsx

```diff
@@ -79,7 +79,7 @@ const HomePage = () => {
     const participantsList = await fetchParticipants({
       sortBy,
       sortDirection,
-      searchSubjects: normalizeSearchSubjects(searchSubjects),
+      searchSubjects: normalizeSearchSubjects(e),
     })
 
     setParticipants(participantsList)
```

Comment:
I noticed in the `useEffect` below we set this after `fetchParticipants` like...

```js
    fetchParticipants().then((participantsList) => {
      const dropDownOptions = participantsList.map(({ study, subject }) => ({
        value: `${subject}`,
        label: `${subject} in ${study}`,
      }))
      setParticipants(participantsList)
      setSearchOptions(dropDownOptions)
      setRowcount(participantsList.length)
    })
```

Do we need to set dropdown options and row count as well?